.storybook/preview-head.html

```diff
@@ -26,18 +26,6 @@
   body * {
     box-sizing: border-box;
   }
-  .width-35 {
-    width: 35%;
-  }
-  .main-story-title {
-    font-size: 32px;
-    color: #323338;
-  }
-  .story-title {
-    color: #323338;
-    font-size: 18px;
-    margin-bottom: 48px;
-  }
```

Comment:
why?

Reply:
Not needed any more - it was used only in storybook-helpers components, which weren't used, so I've deleted them